src/controllers/application.controller.ts

```diff
@@ -5,6 +5,7 @@ import { spawn, execSync } from 'child_process';
 import { existsSync } from 'fs-extra';
 import { InfernodeError } from '../utils/globalErrorHandler';
 import logger from '../utils/logging';
+import { resourceLimits } from 'worker_threads';
```

Comment:
Looks like there is an unused import that can be removed.

Reply:
I have no idea what that is or where it came from. is that not something you added?

Reply:
I don't think so. Maybe a type resulting in an auto import at some point? VSCode can be pretty aggressive about that sometimes.

Reply:
weird. I'll get rid of it.